hedera-node/src/main/java/com/hedera/services/ledger/HederaLedger.java

```diff
@@ -366,13 +366,11 @@ public ResponseCodeEnum doTokenTransfer(
 
 	public void doZeroSum(List<BalanceChange> changes) {
```

Comment:
I removed `final` for the parameter as I was modifying the list. It can be added back now